### ISSUE TITLE
Tailwind Focus Ring isn't working on Codesandbox. Works locally.

### ISSUE THREAD

**Issue by deadcoder0904** [codesandbox/codesandbox-client#5503]
# 🐛 bug report

## Preflight Checklist

- [x] I have read the
      [Contributing Guidelines](https://github.com/codesandbox/codesandbox-client/blob/master/CONTRIBUTING.md)
      for this project.
- [x] I agree to follow the
      [Code of Conduct](https://github.com/codesandbox/codesandbox-client/blob/master/CODE_OF_CONDUCT.md)
      that this project adheres to.
- [x] I have searched the issue tracker for an issue that matches the one I want
      to file, without success.

## Description of the problem

I have a focus ring locally like:

![focus ring tailwind](https://user-images.githubusercontent.com/16436270/108626448-3b9c6400-7476-11eb-8db5-8fcaef08c5d4.png)

I tried to set up a demo project in Tailwind but the focus ring just doesn't work.

### To Reproduce

Just type the following in a Codesandbox with Tailwind:

```html
<div className="w-10 h-10 bg-red-600 rounded-full focus:outline-none ring-4 ring-offset-2 ring-red-500"></div>
```

Or click the link below.

### Link to sandbox: https://codesandbox.io/s/mobx-theme-color-change-p1j6r
